test/lint.js

```diff
@@ -1,10 +0,0 @@
-var lint = require('mocha-eslint');
-
-var paths = [
-  'src/*.js',
-  'src/*/*.js',
```

Comment:
Was this file not used before? I see that `package.json` has a `lint` command:

`"lint": "eslint src/*.js test/*.js"`

I think we should add 'src/*/*.js` and run `yarn lint` to make sure linting is still clean.